File: templates/java/zh/text/code/stepUp2.java
/*
 * 文件：锥体采精锥体.java
 * -------------
 * 锥体采注锥体程序定义了一个带有三个命令的“ run
 * ”方法。这些命令导致卡雷尔向move前进一个
 * 区块，获得锥体 ，然后move前进到下一个角落。
 */
import stanford.卡雷尔.*;
public class 锥体采摘锥体 extends 卡雷尔 {
   public void run() {
      move();
      pickBeeper();
      move();
      turnLeft();
      move();
      turnRight();
      move();
      move();
      putBeeper();
      move();
   }

   /**
    * 卡雷尔向右转90度。
    */
   private void turnRight() {
      turnLeft();
      turnLeft();
      turnLeft();
   }
}

File: templates/python/zh/text/code/stepUp2.py
＃文件：BeeperPickingKarel.py
＃-----------------------------
＃BeeperPickingKarel程序定义了“ main ”
＃函数带有三个命令。这些命令导致
＃卡雷尔（Karel）以move前进一个方块，捡起一个
＃ 锥体 ，然后move到达下一个角。
from 卡雷尔.stanford import *
def main():
   move()
   选择锥体()
   move()
   turn_left()
   move()
   turn_right()
   move()
   move()
   放锥体()
   move()

def turn_right():
   turn_left()
   turn_left()
   turn_left()